app/factories/user.py

```diff
@@ -0,0 +1,27 @@
+import factory
+from app.models.user import db, User
+import app.factories.common as common
+
+
+class UserFactory(factory.alchemy.SQLAlchemyModelFactory):
+    class Meta:
+        model = User
+        sqlalchemy_session = db.session
+
+    email = common.email_
```

Comment:
Just curious to know if we can use **sequence** to generate different unique email IDs ( In case we test it for multiple users with different roles )